test/scripts/transforms/test_osiris_transform.py

```diff
@@ -17,7 +17,7 @@ def generate_input_path_for_run(run_number, cycle):
 # To change by automatic script
 input_runs = [108538, 108539]
 # This needs to be loaded from a shared repository of files
-calibration_run_number = "00148587"
```

Comment:
Another instance of changing a string to a list which is not consistent with the other changes of maintaining a string structure